meta-resin-common/recipes-support/resin-device-api-key/resin-device-api-key/resin-device-api-key

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
```

Comment:
It looks like this is almost identical to resin-device-uuid. Maybe it could be a single script that takes the name of the field (`uuid` or `deviceApiKey`) as an argument, and then each service uses the script passing the corresponding argument?